impeller/aiks/canvas.h

```diff
@@ -37,6 +37,8 @@ struct CanvasStackEntry {
   size_t num_clips = 0u;
   Scalar distributed_opacity = 1.0f;
   Entity::RenderingMode rendering_mode = Entity::RenderingMode::kDirect;
+  // Whether all entities in the current save should be skipped.
+  bool skipping = false;
```

Comment:
When we encounter a saveLayer that should be skipped, we need a way to ensure that no child entities of that saveLayer get rendered. This is a quickish fix to make this work correctly.

In the future, we could change the dispatcher to use indicies and have a more explicit skip step.

Reply:
We could also do a normal `save; clip(Empty)`